tensorflow_addons/optimizers/conditional_gradient_test.py

```diff
@@ -26,7 +27,63 @@ def _update_conditional_gradient_numpy(self, var, norm, g, lr, lambda_):
         var = var * lr - (1 - lr) * lambda_ * g / norm
         return var
 
-    def doTestBasic(self, use_resource=False, use_callable_params=False):
+    def top_singular_vector(self, m):
```

Comment:
You can instead use the method previously defined: `cg_lib.ConditionalGradient._top_singular_vector`. So if any changes are made to the actual impl will reflect here.

Reply:
Got it! Thank you!